packages/11ty/package.json

```
@@ -32,9 +32,10 @@
     "lint:fix": "npm run lint -- --fix",
     "prebuild": "npm run clean",
     "predev": "del-cli public",
+    "prepublishOnly": "npm run test",
     "preserve": "npm run clean",
     "serve": "cross-env ELEVENTY_ENV=production eleventy --serve",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "node --test _plugins/figures/tests/index.js"
```

```suggestion
    "test": "node --test"
```
By defualt `node --test` will run everything in a `test` directory, so if you rename the `11ty/_plugins/figures/tests` directory to `11ty/_plugins/figures/test` and we continue to follow that pattern, this command will run the (future) whole 11ty test suite

---

ah nice!